### ISSUE TITLE
The project does not compile on any version of Flutter as per the documentation in Readme.

### ISSUE THREAD

**Issue by CyberWake** [CCExtractor/beacon#184]
**Describe the bug**
When tried compiling the development/main branch I was unable to get the application up and running. The relevant error thrown was
<img width="1551" alt="Screenshot 2023-02-22 at 1 19 51 AM" src="https://user-images.githubusercontent.com/44184786/220444245-e180e678-0a41-4580-814e-fe2f3ebd1e76.png">

**To Reproduce**

1. Clone the repository
2. Setup flutter on channel stable for any version `3.0.1` or `higher`
3. Add `.env` values

**Expected behavior**
The project should compile as per the Readme with the following flutter versions.

**Environment**
- Device/emulator being used: Pixel 6 Pro API 33(Emulator) iPhone 14 Pro Max(Simulator)
- OS (iOS/ Android): Both
- Flutter version: 
<img width="620" alt="Screenshot 2023-02-22 at 1 24 37 AM" src="https://user-images.githubusercontent.com/44184786/220445115-3f28a294-334e-4205-9593-ff363372a8d1.png">

**Additional context**
This is happening due to the dependency **`graphql_flutter`** used in project with further depends upon `graphql` which is dependent upon `web_socket_channel` and here it it breaking with the current version of `graphql_flutter` used in the beacon project i.e. **`^5.0.0`** 

**Are you working on this issue? (Yes/No)**
**Yes**

**Comment by ItsAdityaKSingh** [comment]
Are these errors for the development branch?

**Comment by CyberWake** [comment]
Yes @ItsAdityaKSingh on `development` also I faced couple of them on `main` branch as well.
